File: aspnet-core/src/group5/Group5.SE347.L11_HelloWork.Application/Services/Educations/Dto/EducationDto.cs
using Abp.Application.Services.Dto;
using Abp.AutoMapper;
using Abp.Events.Bus.Entities;
using SE347.L11_HelloWork.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace Group5.SE347.L11_HelloWork.Application.Services.Educations.Dto
{
    [AutoMapFrom(typeof(Education))]
    public class EducationDto : EntityDto<int>
    {
        public string Name { get; set; }
        public string School { get; set; }
        public DateTime StartYear { get; set; }
        public DateTime EndYear { get; set; }
        public string Majors { get; set; }
        public long? IDJobSeeker { get; set; }
    }
}

File: reactjs/src/app/groups/group5/services/dto/jobSeekerDto/educationDto.ts
export interface CreateEducationInput {
    name: string,
    school: string,
    startYear:Date,
    endYear: Date,
    majors: string,
    idJobSeeker: number,
}

export interface UpdateEducationInput extends CreateEducationInput {
    id: number
}

export interface GetAllEducationByIDOutput<IEducationItem> {
    items: IEducationItem[]
}